src/you_get/extractors/bilibili.py

```diff
@@ -111,6 +112,13 @@ def bilibili_space_video_api(mid, pn=1, ps=100):
     def bilibili_vc_api(video_id):
         return 'https://api.vc.bilibili.com/clip/v1/video/detail?video_id=%s' % video_id
 
+    @staticmethod
+    def url_size(url, faker=False, headers={},err_value=0):
+        try:
```

Comment:
Why is this wrapper function necessary? Under what circumstances can `url_size` from common.py fail?

Reply:
Because sometime the server will return a `403` error for the audio file, but the video file is accessible.